pkgs/development/python-modules/cassandra-driver/default.nix

```diff
@@ -35,6 +35,10 @@ buildPythonPackage rec {
     sha256 = "1dn7iiavsrhh6i9hcyw0mk8j95r5ym0gbrvdca998hx2rnz5ark6";
   };
 
+  postPatch = ''
+    substituteInPlace setup.py --replace 'geomet>=0.1,<0.3' 'geomet'
```

Comment:
Any chances of some version relaxation getting merged upstream?

Reply:
Feel free to open an issue with them telling them all tests seem to pass with 0.3.

Reply:
I did that in https://datastax-oss.atlassian.net/browse/PYTHON-1280, hopefully that's the right place.